notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    os.environ['KACHERY_STORAGE_DIR']="/stelmo/nwb/kachery-storage"
</pre><p>we can probably get rid of this - the user should have specified this in the bashrc anyway if they were following readme. if not, then we should specify all the other environment variables as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>